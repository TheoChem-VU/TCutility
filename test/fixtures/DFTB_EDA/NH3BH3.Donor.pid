# status: ready
# pid: 40706
# hostname: localhost
# username: 
# jobscript: 
# prolog: 
# epilog: 
# jobdir: $HOME/jobs
# fakejob: 0
# comparewithtarget: ok
# error: 0
# warning: 0
# runcmd: sh "$job" "NSCM=$options ; export NSCM"
# plamscmd: local:*,1
# batch: yes
# options: 
# killcmd: killall $pid
# jobstatuscmd: ps -e | grep -w $pid | grep -v grep
# sysstatuscmd: uptime
# pidfile: no
# label: Sequential
# logfile: logfile
# backup: 
# whenready: 
# scmjobid: 1692949746855-19458
# paused: 0
# remoteprefix: 
# jobtype: dftb
# mtime: 1692949751
# cloudinit: 
# cloudid: 
# copyback: 
# privatesshkeypath: 
# comments: 
