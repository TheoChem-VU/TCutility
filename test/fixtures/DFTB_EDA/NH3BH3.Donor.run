#!/bin/sh

"$AMSBIN/ams" << eor

Task SinglePoint
System
    Atoms
        N 4.38957314953014e-29 4.389583116776653e-29 -0.8147415269758744 region=Donor
        H -0.9521670342558302 1.362559924493968e-20 -1.144102945388401 region=Donor
        H 0.4760835121279151 -0.824600842694155 -1.144102945388401 region=Donor
        H 0.4760835121279151 0.824600842694155 -1.144102945388401 region=Donor
    End
    BondOrders
         1 2 1.0
         1 3 1.0
         1 4 1.0
    End
End

Engine DFTB
EndEngine
eor

