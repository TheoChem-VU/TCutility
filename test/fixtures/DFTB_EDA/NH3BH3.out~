

 Parallel Execution: Process Information
 ==============================================================================
 Rank   Node Name                              NodeID   MyNodeRank  NodeMaster
    0   yumans-mbp.home                           0          0          0
    1   yumans-mbp.home                           0          1         -1
    2   yumans-mbp.home                           0          2         -1
    3   yumans-mbp.home                           0          3         -1
    4   yumans-mbp.home                           0          4         -1
    5   yumans-mbp.home                           0          5         -1
    6   yumans-mbp.home                           0          6         -1
    7   yumans-mbp.home                           0          7         -1
    8   yumans-mbp.home                           0          8         -1
    9   yumans-mbp.home                           0          9         -1
 ==============================================================================


May use up to 15625MB of RAM as shared memory on node 0

 *******************************************************************************
 *                                                                             *
 *  --------------------------------                                           *
 *   Amsterdam Modeling Suite (AMS)              2022.103                      *
 *  --------------------------------                                           *
 *                                               r104886 2022-06-17            *
 *                                                                             *
 *                                                                             *
 *                              =================                              *
 *                              |               |                              *
 *                              |     A M S     |                              *
 *                              |               |                              *
 *                              =================                              *
 *                                                                             *
 *                                                                             *
 *   Online information and documentation:  https://www.scm.com/support/       *
 *   E-mail:  support@scm.com   info@scm.com                                   *
 *                                                                             *
 *   Scientific publications using AMS results must be properly referenced     *
 *   See the User Manuals (or the web site) for recommended citations          *
 *   The terms and conditions of the End User License Agreement apply to       *
 *   the use of AMS, https://www.scm.com/license-terms/                        *
 *                                                                             *
 **********************  x86_64_darwin_intel / openmpi  ************************
 
 Licensed to: Prof Visscher, Prof Bickelhaupt, Prof Fonseca Guerra / Vrije Universiteit, Theoretische Chemie / Amsterdam / NETHERLANDS 2022-12-13 09:59:29
 SCM User ID: u21794
  
 AMS 2022.103  RunTime: Aug25-2023 09:44:26  ShM Nodes: 1  Procs: 10

 AMS jobname: ams
 AMS jobid  : 1458786471

 Start   directory: /Users/yumanhordijk/ADF_DATA/dftb/EDA/tmp.yumanhordijk.39782.0.noindex/
 Results directory: /Users/yumanhordijk/ADF_DATA/dftb/EDA/NH3BH3.results/
 Scratch directory: /tmp/amstmp_ams_kid0.1458786471/


 Communication costs MPI_COMM_WORLD:      2.395 usec per message,   0.0239 usec per 8-byte item
 Communication costs for intra-node:      1.831 usec per message,   0.0421 usec per 8-byte item

 RNG seed: 1803278895 490230221 1568051262 -624326427 -2123735709 2039449879 2093528348 -86960286
       ***************************
       *  GEOMETRY OPTIMIZATION  *
       ***************************

Optimization Method                    Quasi Newton
Optimization Coordinates               Delocalized Coordinates
Optimize lattice                       F
 
 ------------------------------------------------------
 Convergence criteria (Hartree,Angstrom)
 ------------------------------------------------------
 Maximum gradient               0.001000000000
 Maximum rms gradient           0.000666666667
 Maximum step allowed           0.010000000000
 Maximum rms step allowed       0.006666666667
 Maximum energy change allowe   0.000080000000
 Maximum stress energy allowe   0.000500000000
 
Initial model Hessian                  Automatic Hessian
 
------------------------------------
Quasi-Newton Optimizer Parameters
------------------------------------
Hessian Update Method                  BFGS
Maximum number of steps                  250
First GDIIS cycle                          0
Maximum GDIIS vectors                      0
Trust radius (bohr)                        0.200000
Trust radius varies                        T
Constraints converged at all steps         T
Use projector                              T
 
 Rigid motions settings for: old optimizer
 Atomic rotations (x,y,z):     T T T
 Atomic translations (x,y,z):  T T T
 Tolerance:                   1.000000000000000E-006
 
--------
Geometry
--------
Formula: H6BN
Atoms
  Index Symbol   x (angstrom)   y (angstrom)   z (angstrom)
      1      N     0.00000000     0.00000000    -0.81493407       region=Donor
      2      B     0.00000000     0.00000000     0.83557436       region=Acceptor
      3      H     0.47602036    -0.82449146    -1.14419799       region=Donor
      4      H     0.47602036     0.82449146    -1.14419799       region=Donor
      5      H    -0.95204073     0.00000000    -1.14419799       region=Donor
      6      H    -0.58148728     1.00716550     1.13731789       region=Acceptor
      7      H    -0.58148728    -1.00716550     1.13731789       region=Acceptor
      8      H     1.16297455     0.00000000     1.13731789       region=Acceptor
 
Total System Charge             0.00000
Bonding info available!
 
 
-------------------------
DFTB Engine configuration
-------------------------
DFTB Resources dir                     /Applications/AMS2022.103.app/Contents/Resources/amshome/atomicdata/DFTB/GFN1-xTB
 
SCC convergence enabled.
Maximum SCC cycles                       500
SCC charge convergence                   0.000000010000
SCC charge mixing                        0.150000000000
DIIS max dimension                       20
DIIS maximum coeff.                     10.000000000000
Adaptive SCC charge mixing             T
Adaptive SCC mixing strategy              3
Spin polarization                      F
Orbital-dependent SCC                  T
Orbital fill strategy                  Fermi
Fermi temperature (kelvin)                     300.0000
  ... kT (Hartree)                               0.0010
Use of symmetry                        T
Radial function extrapolation method   Auto

 Settings for Grimme D3 dispersion correction

      damping                        BJ
      s6                          1.000
      s8                          2.400
      a1                          0.630
      a2                          5.000

 Other (technical) parameters

      alpha                      14.000
      version                         4
 

---------------------
Periodic DFTB options
---------------------
 kpoints config
 grid method with params
Assume insulator                        F
Ewald tolerance                    1.00E-10
Ewald range factor                  2.00

 BZStruct config
 ---------------
 Enabled        T
 Automatic      T
 CalcFatBands   T
 useSymmetry    T
 deltaK          0.100000000000000     
 
Number of symmetry operators: 6
Schoenflies symbol:           C(3V)
 Using automatic Hessian for delocalized coordinates.
 
----------------------------------------------------------------------
Geometry Convergence after Step   1       (Hartree/Angstrom,Angstrom)
----------------------------------------------------------------------
current energy                               -7.82088445 Hartree
energy change                      -7.82088445     0.00008000    F
constrained gradient max            0.00050419     0.00100000    T
constrained gradient rms            0.00014987     0.00066667    T
gradient max                        0.00050419
gradient rms                        0.00014987
cart. step max                      0.00019254     0.01000000    T
cart. step rms                      0.00009838     0.00666667    T
 
 
--------
Geometry
--------
Formula: H6BN
Atoms
  Index Symbol   x (angstrom)   y (angstrom)   z (angstrom)
      1      N     0.00000000     0.00000000    -0.81474153       region=Donor
      2      B    -0.00000000    -0.00000000     0.83567034       region=Acceptor
      3      H     0.47608351    -0.82460084    -1.14410295       region=Donor
      4      H     0.47608351     0.82460084    -1.14410295       region=Donor
      5      H    -0.95216703     0.00000000    -1.14410295       region=Donor
      6      H    -0.58149793     1.00718395     1.13712667       region=Acceptor
      7      H    -0.58149793    -1.00718395     1.13712667       region=Acceptor
      8      H     1.16299585    -0.00000000     1.13712667       region=Acceptor
 
Total System Charge             0.00000
Bonding info available!
 
 
---------------------------------------------------------
Geometry Convergence after Step   2       ** CONVERGED **
---------------------------------------------------------
current energy                               -7.82088458 Hartree
energy change                      -0.00000013     0.00008000    T
constrained gradient max            0.00019587     0.00100000    T
constrained gradient rms            0.00004755     0.00066667    T
gradient max                        0.00019587
gradient rms                        0.00004755
cart. step max                      0.00008829     0.01000000    T
cart. step rms                      0.00002383     0.00666667    T
 
Geometry optimization converged

Optimized geometry:
 
--------
Geometry
--------
Formula: H6BN
Atoms
  Index Symbol   x (angstrom)   y (angstrom)   z (angstrom)
      1      N     0.00000000     0.00000000    -0.81474153       region=Donor
      2      B    -0.00000000    -0.00000000     0.83567034       region=Acceptor
      3      H     0.47608351    -0.82460084    -1.14410295       region=Donor
      4      H     0.47608351     0.82460084    -1.14410295       region=Donor
      5      H    -0.95216703     0.00000000    -1.14410295       region=Donor
      6      H    -0.58149793     1.00718395     1.13712667       region=Acceptor
      7      H    -0.58149793    -1.00718395     1.13712667       region=Acceptor
      8      H     1.16299585    -0.00000000     1.13712667       region=Acceptor
 
Total System Charge             0.00000
Bonding info available!
 
Performing a single point calculation on the final geometry ...

Estimated upper bound for memory consumption:    0.065 MiB

Number of basis functions:           20
    Cycle:   1, MaxQDiff:   0.000000000004, PartialEnergy:      -7.9128503806, Used mix:   0.150 nvec:    0
 Evaluation was successful


    =====================
     CALCULATION RESULTS
    =====================

Energy (hartree)            -7.82088458

Gradients (hartree/bohr)
  Index   Atom            d/dx            d/dy            d/dz
      1      N    0.0000000059   -0.0000000000   -0.0001036489
      2      B   -0.0000000019   -0.0000000000    0.0000090610
      3      H   -0.0000093379    0.0000161695    0.0000221639
      4      H   -0.0000093379   -0.0000161695    0.0000221639
      5      H    0.0000186700    0.0000000000    0.0000221634
      6      H   -0.0000114978    0.0000199147    0.0000093656
      7      H   -0.0000114978   -0.0000199147    0.0000093656
      8      H    0.0000229974    0.0000000000    0.0000093655


--------------
Atomic charges
--------------
  Index   Atom    Charge
      1      N   -0.3146
      2      B    0.0333
      3      H    0.2232
      4      H    0.2232
      5      H    0.2232
      6      H   -0.1295
      7      H   -0.1295
      8      H   -0.1295


Dipole moment vector    (Debye):    -0.00000000    0.00000000   -4.43608394
              magnitude (Debye):     4.43608394

 
--------------------
Energy Decomposition
--------------------
Total Energy (hartree)                -7.82088458
Total Energy (eV)                   -212.81709753
Electronic Energy (hartree)           -7.93826672
Coulomb Energy (hartree)               0.02541634
Repulsion Energy (hartree)             0.09334726
Dispersion Energy (hartree)           -0.00138146
 
----------------------
Gradient Decomposition
----------------------
Total Gradients (hartree/bohr)
  Index   Atom            d/dx            d/dy            d/dz
      1      N    0.0000000059   -0.0000000000   -0.0001036489
      2      B   -0.0000000019   -0.0000000000    0.0000090610
      3      H   -0.0000093379    0.0000161695    0.0000221639
      4      H   -0.0000093379   -0.0000161695    0.0000221639
      5      H    0.0000186700    0.0000000000    0.0000221634
      6      H   -0.0000114978    0.0000199147    0.0000093656
      7      H   -0.0000114978   -0.0000199147    0.0000093656
      8      H    0.0000229974    0.0000000000    0.0000093655
 
Total Gradients (eV/angstrom)
  Index   Atom            d/dx            d/dy            d/dz
      1      N    0.0000003036   -0.0000000000   -0.0053298425
      2      B   -0.0000000990   -0.0000000000    0.0004659374
      3      H   -0.0004801719    0.0008314687    0.0011397114
      4      H   -0.0004801719   -0.0008314687    0.0011397114
      5      H    0.0009600507    0.0000000000    0.0011396889
      6      H   -0.0005912420    0.0010240541    0.0004815988
      7      H   -0.0005912420   -0.0010240541    0.0004815988
      8      H    0.0011825724    0.0000000000    0.0004815959
 
Electronic Gradients (hartree/bohr)
  Index   Atom            d/dx            d/dy            d/dz
      1      N    0.0000000064   -0.0000000000    0.0590756431
      2      B   -0.0000000012   -0.0000000000   -0.0156104132
      3      H    0.0402859312   -0.0697772847   -0.0242653887
      4      H    0.0402859312    0.0697772847   -0.0242653887
      5      H   -0.0805718686    0.0000000000   -0.0242653891
      6      H   -0.0211136349    0.0365698878    0.0097769789
      7      H   -0.0211136349   -0.0365698878    0.0097769789
      8      H    0.0422272708    0.0000000000    0.0097769788
 
Coulomb Gradients (hartree/bohr)
  Index   Atom            d/dx            d/dy            d/dz
      1      N    0.0000000000    0.0000000000    0.0099675906
      2      B    0.0000000000    0.0000000000   -0.0002991916
      3      H    0.0007673919   -0.0013291618   -0.0041130491
      4      H    0.0007673919    0.0013291618   -0.0041130491
      5      H   -0.0015347838    0.0000000000   -0.0041130491
      6      H    0.0005258535   -0.0009108050    0.0008902495
      7      H    0.0005258535    0.0009108050    0.0008902495
      8      H   -0.0010517070   -0.0000000000    0.0008902494
 
Repulsion Gradients (hartree/bohr)
  Index   Atom            d/dx            d/dy            d/dz
      1      N   -0.0000000005    0.0000000000   -0.0691541531
      2      B   -0.0000000007   -0.0000000000    0.0159134178
      3      H   -0.0410652704    0.0711271357    0.0284105418
      4      H   -0.0410652704   -0.0711271357    0.0284105418
      5      H    0.0821305413   -0.0000000000    0.0284105417
      6      H    0.0205695266   -0.0356274648   -0.0106636300
      7      H    0.0205695266    0.0356274648   -0.0106636300
      8      H   -0.0411390526    0.0000000000   -0.0106636299
 
Dispersion Gradients (hartree/bohr)
  Index   Atom            d/dx            d/dy            d/dz
      1      N   -0.0000000000    0.0000000000    0.0000072704
      2      B    0.0000000000   -0.0000000000    0.0000052481
      3      H    0.0000026094   -0.0000045197   -0.0000099401
      4      H    0.0000026094    0.0000045197   -0.0000099401
      5      H   -0.0000052189    0.0000000000   -0.0000099401
      6      H    0.0000067569   -0.0000117033    0.0000057672
      7      H    0.0000067569    0.0000117033    0.0000057672
      8      H   -0.0000135138    0.0000000000    0.0000057672
 
----------------
Mulliken Charges
----------------
  Index   Atom          Charge      Population
      1      N   -0.3145874995        5.314587
      2      B    0.0333416190        2.966658
      3      H    0.2232056316        0.776794
      4      H    0.2232056316        0.776794
      5      H    0.2232056312        0.776794
      6      H   -0.1294570047        1.129457
      7      H   -0.1294570047        1.129457
      8      H   -0.1294570044        1.129457
         Total    0.0000000000       14.000000
 
----------------------
Mulliken Shell Charges
----------------------
  Index   Atom      L          Charge      Population
      1      N      0    0.5462243021        1.453776
      2      N      1   -0.8608118016        3.860812
      3      B      0    1.1214401093        0.878560
      4      B      1   -1.0880984904        2.088098
      5      H      0    0.2465278935        0.753472
      6      H      0   -0.0233222619        0.023322
      7      H      0    0.2465278935        0.753472
      8      H      0   -0.0233222619        0.023322
      9      H      0    0.2465278932        0.753472
     10      H      0   -0.0233222620        0.023322
     11      H      0   -0.1248456588        1.124846
     12      H      0   -0.0046113460        0.004611
     13      H      0   -0.1248456588        1.124846
     14      H      0   -0.0046113460        0.004611
     15      H      0   -0.1248456586        1.124846
     16      H      0   -0.0046113458        0.004611
                Total    0.0000000000       14.000000
 
 
--------
Orbitals
--------
Index of HOMO                    7
Fermi Energy (hartree)           -0.30115760
Orbitals
   Index  Energy (hartree)       Energy (eV)        Occupation  Degeneracy
       1       -0.81903521      -22.28708205        2.00000000           1
       2       -0.60833584      -16.55366059        2.00000000           2
       3       -0.60833584      -16.55366057        2.00000000           2
       4       -0.56046817      -15.25111488        2.00000000           1
       5       -0.46821380      -12.74074577        2.00000000           1
       6       -0.40227392      -10.94643035        2.00000000           2
       7       -0.40227392      -10.94643033        2.00000000           2
       8       -0.17932877       -4.87978416        0.00000000           1
       9       -0.10112896       -2.75185900        0.00000000           2
      10       -0.10112896       -2.75185899        0.00000000           2
      11       -0.03847001       -1.04682217        0.00000000           1
      12       -0.00112805       -0.03069592        0.00000000           2
      13       -0.00112805       -0.03069591        0.00000000           2
      14        0.05141884        1.39917784        0.00000000           1
      15        0.21748906        5.91817846        0.00000000           2
      16        0.21748907        5.91817859        0.00000000           2
      17        0.30409289        8.27478851        0.00000000           1
      18        0.35788858        9.73864389        0.00000000           2
      19        0.35788859        9.73864406        0.00000000           2
      20        0.65219229       17.74705517        0.00000000           1
 

AMS application finished. Exiting.
NORMAL TERMINATION
     Total cpu time:       1.02
  Total system time:       0.70
 Total elapsed time:       2.16
