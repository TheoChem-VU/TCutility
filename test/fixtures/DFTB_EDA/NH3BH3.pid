# status: ready
# pid: 40578
# hostname: localhost
# username: 
# jobscript: 
# prolog: 
# epilog: 
# jobdir: $HOME/jobs
# fakejob: 0
# comparewithtarget: ok
# error: 0
# warning: 0
# runcmd: sh "$job" "NSCM=$options ; export NSCM"
# plamscmd: local:*,1
# batch: yes
# options: 
# killcmd: killall $pid
# jobstatuscmd: ps -e | grep -w $pid | grep -v grep
# sysstatuscmd: uptime
# pidfile: no
# label: Sequential
# logfile: logfile
# backup: 
# whenready: 
# scmjobid: 1692949463867-36503
# paused: 0
# remoteprefix: 
# jobtype: dftb
# mtime: 1692949760
# cloudinit: 
# cloudid: 
# copyback: 
# privatesshkeypath: 
# comments: 
# dependency: start /Users/yumanhordijk/ADF_DATA/dftb/EDA/NH3BH3.Acceptor.job
# dependency: start /Users/yumanhordijk/ADF_DATA/dftb/EDA/NH3BH3.Donor.job
