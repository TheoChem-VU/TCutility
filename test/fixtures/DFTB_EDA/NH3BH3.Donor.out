

 Parallel Execution: Process Information
 ==============================================================================
 Rank   Node Name                              NodeID   MyNodeRank  NodeMaster
    0   yumans-mbp.home                           0          0          0
    1   yumans-mbp.home                           0          1         -1
    2   yumans-mbp.home                           0          2         -1
    3   yumans-mbp.home                           0          3         -1
    4   yumans-mbp.home                           0          4         -1
    5   yumans-mbp.home                           0          5         -1
    6   yumans-mbp.home                           0          6         -1
    7   yumans-mbp.home                           0          7         -1
    8   yumans-mbp.home                           0          8         -1
    9   yumans-mbp.home                           0          9         -1
 ==============================================================================


May use up to 15625MB of RAM as shared memory on node 0

 *******************************************************************************
 *                                                                             *
 *  --------------------------------                                           *
 *   Amsterdam Modeling Suite (AMS)              2022.103                      *
 *  --------------------------------                                           *
 *                                               r104886 2022-06-17            *
 *                                                                             *
 *                                                                             *
 *                              =================                              *
 *                              |               |                              *
 *                              |     A M S     |                              *
 *                              |               |                              *
 *                              =================                              *
 *                                                                             *
 *                                                                             *
 *   Online information and documentation:  https://www.scm.com/support/       *
 *   E-mail:  support@scm.com   info@scm.com                                   *
 *                                                                             *
 *   Scientific publications using AMS results must be properly referenced     *
 *   See the User Manuals (or the web site) for recommended citations          *
 *   The terms and conditions of the End User License Agreement apply to       *
 *   the use of AMS, https://www.scm.com/license-terms/                        *
 *                                                                             *
 **********************  x86_64_darwin_intel / openmpi  ************************
 
 Licensed to: Prof Visscher, Prof Bickelhaupt, Prof Fonseca Guerra / Vrije Universiteit, Theoretische Chemie / Amsterdam / NETHERLANDS 2022-12-13 09:59:29
 SCM User ID: u21794
  
 AMS 2022.103  RunTime: Aug25-2023 09:49:15  ShM Nodes: 1  Procs: 10

 AMS jobname: ams
 AMS jobid  : 1714475311

 Start   directory: /Users/yumanhordijk/ADF_DATA/dftb/EDA/tmp.yumanhordijk.40706.0.noindex/
 Results directory: /Users/yumanhordijk/ADF_DATA/dftb/EDA/NH3BH3.Donor.results/
 Scratch directory: /tmp/amstmp_ams_kid0.1714475311/


 Communication costs MPI_COMM_WORLD:      6.389 usec per message,   0.0498 usec per 8-byte item
 Communication costs for intra-node:      3.157 usec per message,   0.0392 usec per 8-byte item

 RNG seed: 1656653578 -1255304810 -153073037 1327855454 240722993 -1489095300 1937639616 -1520473053
       ****************************
       * SINGLE POINT CALCULATION *
       ****************************
 
--------
Geometry
--------
Formula: H3N
Atoms
  Index Symbol   x (angstrom)   y (angstrom)   z (angstrom)
      1      N     0.00000000     0.00000000    -0.81474153       region=Donor
      2      H    -0.95216703     0.00000000    -1.14410295       region=Donor
      3      H     0.47608351    -0.82460084    -1.14410295       region=Donor
      4      H     0.47608351     0.82460084    -1.14410295       region=Donor
 
Total System Charge             0.00000
Bonding info available!
 
 
-------------------------
DFTB Engine configuration
-------------------------
DFTB Resources dir                     /Applications/AMS2022.103.app/Contents/Resources/amshome/atomicdata/DFTB/GFN1-xTB
 
SCC convergence enabled.
Maximum SCC cycles                       500
SCC charge convergence                   0.000000010000
SCC charge mixing                        0.150000000000
DIIS max dimension                       20
DIIS maximum coeff.                     10.000000000000
Adaptive SCC charge mixing             T
Adaptive SCC mixing strategy              3
Spin polarization                      F
Orbital-dependent SCC                  T
Orbital fill strategy                  Fermi
Fermi temperature (kelvin)                     300.0000
  ... kT (Hartree)                               0.0010
Use of symmetry                        T
Radial function extrapolation method   Auto

 Settings for Grimme D3 dispersion correction

      damping                        BJ
      s6                          1.000
      s8                          2.400
      a1                          0.630
      a2                          5.000

 Other (technical) parameters

      alpha                      14.000
      version                         4
 

---------------------
Periodic DFTB options
---------------------
 kpoints config
 grid method with params
Assume insulator                        F
Ewald tolerance                    1.00E-10
Ewald range factor                  2.00

 BZStruct config
 ---------------
 Enabled        T
 Automatic      T
 CalcFatBands   T
 useSymmetry    T
 deltaK          0.100000000000000     
 
Estimated upper bound for memory consumption:    0.024 MiB

Number of basis functions:           10
    Cycle:   1, MaxQDiff:   1.395524809287, PartialEnergy:      -4.8671101165, Used mix:   0.150 nvec:    0
    Cycle:   2, MaxQDiff:   1.261216432061, PartialEnergy:      -4.8711633848, Used mix:   0.150 nvec:    1
    Cycle:   3, MaxQDiff:   1.019578031896, PartialEnergy:      -4.8770875163, Used mix:   0.150 nvec:    1
    Cycle:   4, MaxQDiff:   0.169808212935, PartialEnergy:      -4.8849799372, Used mix:   0.150 nvec:    2
    Cycle:   5, MaxQDiff:   0.137124370815, PartialEnergy:      -4.8851056861, Used mix:   0.150 nvec:    2
    Cycle:   6, MaxQDiff:   0.000042789157, PartialEnergy:      -4.8853973375, Used mix:   0.150 nvec:    3
    Cycle:   7, MaxQDiff:   0.000046846272, PartialEnergy:      -4.8853973375, Used mix:   0.150 nvec:    4
    Cycle:   8, MaxQDiff:   0.000000409775, PartialEnergy:      -4.8853973375, Used mix:   0.150 nvec:    4
    Cycle:   9, MaxQDiff:   0.000000000028, PartialEnergy:      -4.8853973375, Used mix:   0.150 nvec:    4
 Evaluation was successful


    =====================
     CALCULATION RESULTS
    =====================

Energy (hartree)            -4.82864553


--------------
Atomic charges
--------------
  Index   Atom    Charge
      1      N   -0.5797
      2      H    0.1932
      3      H    0.1932
      4      H    0.1932


Dipole moment vector    (Debye):    -0.00000001    0.00000000   -0.91703086
              magnitude (Debye):     0.91703086

 
--------------------
Energy Decomposition
--------------------
Total Energy (hartree)                -4.82864553
Total Energy (eV)                   -131.39413034
Electronic Energy (hartree)           -4.90609143
Coulomb Energy (hartree)               0.02069409
Repulsion Energy (hartree)             0.05704432
Dispersion Energy (hartree)           -0.00029251
 
----------------
Mulliken Charges
----------------
  Index   Atom          Charge      Population
      1      N   -0.5796691094        5.579669
      2      H    0.1932230361        0.806777
      3      H    0.1932230367        0.806777
      4      H    0.1932230367        0.806777
         Total    0.0000000000        8.000000
 
----------------------
Mulliken Shell Charges
----------------------
  Index   Atom      L          Charge      Population
      1      N      0    0.5635807752        1.436419
      2      N      1   -1.1432498846        4.143250
      3      H      0    0.2088133365        0.791187
      4      H      0   -0.0155903004        0.015590
      5      H      0    0.2088133369        0.791187
      6      H      0   -0.0155903002        0.015590
      7      H      0    0.2088133369        0.791187
      8      H      0   -0.0155903002        0.015590
                Total    0.0000000000        8.000000
 
 
--------
Orbitals
--------
Index of HOMO                    4
Fermi Energy (hartree)           -0.36908809
Orbitals
   Index  Energy (hartree)       Energy (eV)        Occupation  Degeneracy
       1       -0.73722514      -20.06091667        2.00000000           1
       2       -0.53233829      -14.48566202        2.00000000           2
       3       -0.53233829      -14.48566199        2.00000000           2
       4       -0.40740727      -11.08611592        2.00000000           1
       5       -0.12905659       -3.51180856        0.00000000           1
       6       -0.04224483       -1.14954027        0.00000000           2
       7       -0.04224483       -1.14954026        0.00000000           2
       8        0.29142899        7.93018622        0.00000000           2
       9        0.29142899        7.93018636        0.00000000           2
      10        0.67883488       18.47203705        0.00000000           1
 

AMS application finished. Exiting.
NORMAL TERMINATION
     Total cpu time:       1.04
  Total system time:       0.60
 Total elapsed time:       2.14
